resource/airflow_dag_pipeline.yaml

```diff
@@ -0,0 +1,79 @@
+---
+jobs:
+- name: deploy
+  plan:
+
+  - get: release
+    trigger: true
+    params:
+      include_source_tarball: true
+
+  - get: common-tasks
+
+  - task: dag-source
+    file: common-tasks/extract-release-tarball.yaml
+    output_mapping: {extracted: dag-source}
+
+  - task: deploy-params
+    file: common-tasks/parse-deploy-file.yaml
+    input_mapping: {webapp-source: dag-source}
+    params:
+      IPS_DOM1: ((secrets.ip-dom1))
+      IPS_QUANTUM: ((secrets.ip-quantum))
+      IPS_102PF_WIFI: ((secrets.ip-102pf))
+
+  - put: dag-docker-repo
+
+  - task: dag-docker-image
+    privileged: true
+    file: common-tasks/webapp-docker-image.yaml
+    input_mapping:
+      webapp-docker-repo: dag-docker-repo
+      webapp-source: dag-source
+    params:
+      AWS_ACCESS_KEY_ID: ((secrets.ecr-access-key-id))
+      AWS_SECRET_ACCESS_KEY: ((secrets.ecr-secret-access-key))
+
+  - task: create-iam-role-name
+    file: common-tasks/put-iam-role.yaml
+
+    output_mapping: {output: aws}
+    input_mapping:
+      source: dag-source
+
+    params:
+      ORG_NAME: ((github-org))
+      REPO_NAME: ((github-repo))
+      AWS_ACCESS_KEY_ID: ((secrets.role-putter-access-key-id))
+      AWS_SECRET_ACCESS_KEY: ((secrets.role-putter-secret-access-key))
+      AWS_NODES_ROLE_ARN: ((secrets.nodes-role-arn))
+      APP_NAME: ((app-name))
+
+resources:
+- name: common-tasks
+  type: git
+  source:
+    uri: https://github.com/ministryofjustice/analytics-platform-common-concourse-tasks.git
+    branch: support-dag-pipelines
```

Comment:
once https://github.com/ministryofjustice/analytics-platform-common-concourse-tasks/pull/7 is merged we need to update this to remove line number 57 so that it uses the master branch again